ecs-cli/modules/cli/compose/project/project_test.go

```diff
@@ -17,12 +17,15 @@ import (
 	"flag"
 	"io/ioutil"
 	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/context"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/compose"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/regcredio"
+	lConfig "github.com/docker/libcompose/config"
```

Comment:
Hm, i think the convention is to use all lowercase for renaming import declarations? is there a reason this needs to be renamed btw?

Update: i see this is copy pasta from elsewhere. something to think about still though.